### ISSUE TITLE
command report issue

### ISSUE THREAD

**Issue by M99BE** [KazzyAPI/steam-report-bot#1]
![image](https://user-images.githubusercontent.com/49033300/151842924-8a77c80d-9742-4f7f-a886-ccfde9d68138.png)
bot ignoring my report commands, console is silent too.

**Comment by KazzyAPI** [comment]
Can you screenshot console when you run a command ?

**Comment by M99BE** [comment]
> Can you screenshot console when you run a command ?
![image](https://user-images.githubusercontent.com/49033300/151850119-5c89404e-c8fc-423e-9dac-6a7ee9a32b3c.png)

**Comment by codsec** [comment]
did u found a fix?

**Comment by M99BE** [comment]
> did u found a fix?
 
nope,but i found anonether one reportbot, and lost all used accounts(20 0lvl accounts)

**Comment by codsec** [comment]
> > did u found a fix?
> 
> nope,but i found anonether one reportbot, and lost all used accounts(20 0lvl accounts)

can u send me that other report bot?

**Comment by KazzyAPI** [comment]
I'm working on a new report bot which will be open source and much more user friendly for you guys. Will release a server for you guys to add suggestions

**Comment by KazzyAPI** [comment]
https://discord.gg/qUe5zFARZw
Nothing going on yet but i hope to get you guys in to see what we can do

**Comment by codsec** [comment]
> I'm working on a new report bot which will be open source and much more user friendly for you guys. Will release a server for you guys to add suggestions

any prediction of when u are gonna release it?
